java/tools/pom.xml

```diff
@@ -62,7 +66,7 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-hdfs</artifactId>
+      <artifactId>hadoop-hdfs-client</artifactId>
```

Comment:
After ORC-603 (https://github.com/apache/orc/pull/607), we use `hadoop-hdfs-client`.

Reply:
Thank you for catching this, @williamhyun .